src/components/CurrencyModal.js

```diff
@@ -1,36 +0,0 @@
-import React, { Component } from 'react';
```

Comment:
I removed this component from the project as it was never actually in use and not needed.